src/main/client_context.cpp

```diff
@@ -390,7 +391,19 @@ std::vector<std::shared_ptr<Statement>> ClientContext::parseQuery(std::string_vi
         transactionContext->beginAutoTransaction(true /* readOnlyStatement */);
     }
     try {
-        statements = Parser::parseQuery(query, this);
+        auto parsedStatements = Parser::parseQuery(query, this);
+        StandaloneCallAnalyzer standaloneCallAnalyzer{this};
+        for (auto i = 0u; i < parsedStatements.size(); i++) {
```

Comment:
Let's talk about this too. @andyfengHKU should review it but I also want to understand what's happening here.